src/phoenix/db/migrations/versions/cf03bd6bae1d_init.py

```diff
@@ -77,6 +77,41 @@ def upgrade() -> None:
         sa.Column("cumulative_llm_token_count_prompt", sa.Integer, nullable=False),
         sa.Column("cumulative_llm_token_count_completion", sa.Integer, nullable=False),
     )
+
+    op.create_table(
+        "span_annotations",
+        sa.Column("id", sa.Integer, primary_key=True),
+        sa.Column("span_rowid", sa.Integer, sa.ForeignKey("traces.id"), nullable=False),
+        sa.Column("name", sa.String, nullable=False),
+        sa.Column("label", sa.String, nullable=True),
+        sa.Column("score", sa.Float, nullable=True),
+        sa.Column("explanation", sa.String, nullable=True),
+        sa.Column("metadata", sa.JSON, nullable=False),
+        sa.Column(
+            "annotator_kind",
+            sa.String,
+            nullable=False,
+        ),
+        sa.Column(
+            "created_at",
+            sa.DateTime(timezone=True),
+            nullable=False,
+            server_default=sa.func.now(),
+        ),
+        sa.Column(
+            "updated_at",
+            sa.DateTime(timezone=True),
+            nullable=False,
+            server_default=sa.func.now(),
+            onupdate=sa.func.now(),
+        ),
+        sa.UniqueConstraint("span_rowid", "name", name="uq_span_annotations_span_rowid_name"),
+        sa.CheckConstraint(
+            "annotator_kind IN ('LLM', 'HUMAN')",
+            name="valid_annotator_kind",
+        ),
+    )
```

Comment:
can't you place this directly on the column?

Reply:
addressed in 43e926b11e5a43f97efb1abcfea7eb3cfd7d4ab2